### ISSUE TITLE
Annotations Targeting Multiple Manifests

### ISSUE THREAD

**Issue by cubap** [IIIF/cookbook-recipes#193]
## Annotations Targeting Multiple Manifests  

(Provide link to `index.md` of the issue, if available. To be filled in after issue is created - you need the issue number!)

## Use case  
In a curated collection, there are multiple witnesses annotated with the same theme or transcription. These may be separate Canvases in the same Manifest (which is already covered), but may also be separate Manifests in (or not in) a Collection or independent resources. The Annotation creates the link between them following the Web Annotation case for multiple `target`s. These cases are unlikely to reveal their additional data in generic viewers, but there is still a good reason to present them this way.

**Comment by cubap** [comment]
In a case currently implemented, these Annotations may also cross resource types outside of current coverage in the A/V scope. At [Sounding Tennyson](https://www.soundingtennyson.org/app.html#/manuscript/AET-MSS-5321-001), a performance is synced with the display of Canvases in several Manifests (as well as a text resource). The existing implementation predates and does not leverage Presi3, but still escapes its scope.

The Annotations do not need to be within the targeted resource by the content owner, but probably would be shown that way for this recipe.

**Comment by glenrobson** [comment]
This seems a difficult one to write at this current time. Particularly as it seems difficult to create these annotations without creating them by hand. Also we don't think we've seen generic viewers that would display these types of annotations in a generic way. So there is a question if this can be a recipe that can be re-used by others. 

It would be good to have more examples of others doing this so we can see the commonality.

**Comment by cubap** [comment]
> This seems a difficult one to write at this current time. ...
> It would be good to have more examples of others doing this so we can see the commonality.

I agree. In our instance, we needed all the `target`s and `body`s together, but I can imagine that some management system would let a user tie transcriptions to several different Manifests that it serves and only deliver each Manifest with internal targets.

I would like to know who else or how else this is being done, as I think it would be a new level of interaction between the IIIF and Web Annotation standards that would necessarily create more LOUD instances in the wild.
